StorageUtility/Linux/StorageUtility.cpp

```diff
@@ -210,4 +211,11 @@ void CloseDeviceHandle(DeviceHandle& Handle)
     }
 }
 
+eErrorCode GetPhysicalDiskNumber(DeviceHandle Handle, U32& PhysicalDiskNumber)
+{
+    //! TODO Use Device path on Linux system instead PhysicalDiskNumber
+
+    return(eErrorCode::None);
```

Comment:
It isn't this function's problem. This function doesn't have responsibility to get the code in DriveEnumeratorAta/Scsi works. It should be changed inside them not this.